e2e/utils/test-utils.ts

```diff
@@ -501,3 +501,11 @@ export async function createDatasetNotebook(page: Page, pyNotebookName: string):
 export function parseForNumericalStrings(text: string): RegExpMatchArray | null {
   return text.match(/(\d|\.)+/g);
 }
+
+// Find and delete the workspace
+export async function cleanupWorkspace(page: Page, workspaceName: string) {
+  const workspaceCard = await findWorkspaceCard(page, workspaceName);
```

Comment:
The test still succeeds if the workspace does not exist 👍 

But the test fails if it encounters an error while deleting the workspace: we should add a `catch` here or similar.

Reply:
Looks good now, thanks!